app/controllers/DefaultsController.scala

```diff
@@ -41,9 +41,11 @@ class DefaultsController(val acl: Acl, val isDev: Boolean, val deps: BaseFaciaCo
   def configuration = AccessAPIAuthAction { implicit request =>
     val hasBreakingNews = acl.testUser(Permissions.BreakingNewsAlert, "facia-tool-allow-breaking-news-for-all")(request.user.email)
     val hasConfigureFronts = acl.testUser(Permissions.ConfigureFronts, "facia-tool-allow-config-for-all")(request.user.email)
+    val hasEditionsPermissions = acl.testUser(Permissions.EditEditions, "facia-tool-allow-edit-editorial-fronts-for-all")(request.user.email)
```

Comment:
Non-blocking: is this the right name for a potential editions switch? Perhaps better as 
```suggestion
    val hasEditionsPermissions = acl.testUser(Permissions.EditEditions, "facia-tool-allow-edit-editions-for-all")(request.user.email)
```

There's no scenario I can think of where we'd need to turn this off for every user, but as it stands this referencing editorial fronts, not editions, might be confusing to developers.

Reply:
Makes sense I will change to suggested one 👍